function-images/tests/chained-function-serving/consumer/consumer.go

```diff
@@ -42,13 +43,17 @@ type consumerServer struct {
 	pb.UnimplementedProducerConsumerServer
 }
 
+var tracingEnabled = os.Getenv("TRACING") != ""
```

Comment:
rename: ENABLE_TRACING

Reply:
make the values to be "true" or "false", opposed to "". Harder to make mistakes

add a call IstracingEnabled() to the tracing module. the tracing module should carefully check the validity of the env var's values (only true and false are valid values, log.Fatal() if invalid)